src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -68,6 +74,26 @@ public void onHungerDamage(EntityDamageEvent e) {
         }
     }
 
+    @EventHandler
+    public void onBlockBreak(BlockBreakEvent e) {
+        Player p = e.getPlayer();
+        if (Constants.MACHINE_BREAK_REQUIRES_WRENCH
+            && (!Items.WRENCH.isSimilar(p.getInventory().getItemInMainHand()))
+        ) {
+            Block block = e.getBlock();
+            SlimefunItem slimefunBlock = BlockStorage.check(block);
+
+            if (slimefunBlock instanceof EnergyNetComponent
+                && !slimefunBlock.getID().endsWith("_CAPACITOR")
+            ) {
+                e.setCancelled(true);
+                Wrench.dropBlock(e, p, block, slimefunBlock, true);
+                p.sendMessage(ChatColor.RED + "You need a Wrench to break Slimefun machines!");
+                p.sendMessage(ChatColor.RED + "(Slimefun Guide > LiteXpansion > Wrench)");
```

Comment:
Should also add a LiteXpansion prefix to these messages.

Reply:
We should have a somesort of message.yml